lightning/Cargo.toml

```diff
@@ -24,7 +24,7 @@ max_level_trace = []
 # Allow signing of local transactions that may have been revoked or will be revoked, for functional testing (e.g. justice tx handling).
 # This is unsafe to use in production because it may result in the counterparty publishing taking our funds.
 unsafe_revoked_tx_signing = []
-unstable = []
+_bench_unstable = []
```

Comment:
Maybe just call this `_bench`? Presumably we'd still want this when no longer unstable.

Reply:
Well the only reason we have a feature is because its unstable. If `cargo bench` weren't unstable we'd remove the feature entirely (though there's some indication `cargo bench` will never be stabilized).